gateway/api/models_proxies.py

```diff
@@ -0,0 +1,76 @@
+"""Proxies for database models"""
+
+import requests
+
+from django.conf import settings
+from django.contrib.auth import get_user_model
+
+from api.utils import safe_request
+
+
+class QuantumUserProxy(get_user_model()):
```

Comment:
I think in a near future we could even refactorize `authentication` to include login and validation logic in this proxy.